README.md

```diff
@@ -45,6 +45,7 @@ All these currently assume you have a <samp>&lt;user></samp> login with sudo rig
 
    ```shell
    micromamba create -n asv -c conda-forge conda mamba virtualenv asv
+   micromamba activate asv
    micromamba run -n asv asv machine --yes
    ```
```

Comment:
`micromamba run -n asv` specifies the environment, you don’t need to activate it first. 

````suggestion
   micromamba run -n asv asv machine --yes
   ```

   (use `micromamba activate asv` to make `asv` available in your PATH)
````